docs/source/demo.md

```diff
@@ -1,67 +1,101 @@
 # Testing on the Demo Network
 
-Livepeer is pleased to offer a demo network that can be used to evaluate the Livepeer Transcoding API. This demo network will transcode a live RTMP input stream to a live HLS output stream. The HLS playlist will contain the source stream and two transcoded renditions: 352x240 at 30fps, and 480x360 at 30fps. The HLS output can be used for Adaptive Bitrate Streaming, and is compatible with video players that follows the HLS specification.
+Livepeer is pleased to offer a demo network that can be used to evaluate the Livepeer Transcoding API. This demo network will transcode a live RTMP input stream to a live HLS output stream. The HLS playlist will contain the source stream and two transcoded renditions: `352x240` at `30fps`, and `480x360` at `30fps`. The HLS output can be used for Adaptive Bitrate Streaming, and is compatible with video players that follows the HLS specification.
 
 Please [contact us](mailto:beta@liveeer.org?subject=Ingest+URL+Provision+Request) to apply for access to the demo network.
 
-## 1. Obtain an ingest server URL from Livepeer
 
-Livepeer will give you a set of RTMP ingest and HLS output URLs. The `manifestID` parameter in the ingest URL can be used to form the output URL. For example, when the ingest URL is:
+## Setup: obtain an ingest server URL from Livepeer
 
+Livepeer will give you a set of RTMP ingest, HLS output, and status URLs. The `manifestID` parameter in the ingest URL can be used to form the output URL. 
+
+**For example:**
+The ingest URL can be:
 ```
-rtmp://ingest-host.livepeer.org/?manifestID=customerName_streamName
+rtmp://35.231.43.235:2281/?manifestID=stream1
```

Comment:
I'm not sure we should be publicizing real URLs like this. The demo network has a finite capacity and isn't meant to be a free-for-all.

Reply:
Agree.  I reformatted the doc to take care of this.